script_runner.py

```diff
@@ -19,7 +19,11 @@ def __init__(self, job_id, shippable_adapter, \
         self.console_buffer = []
         self.console_buffer_lock = threading.Lock()
         self.continue_trigger_flush_console_output = True
-        self.max_consoles_size_in_bytes = 64 * 1024 * 1024  # 64 MB
+        self.max_consoles_size_MB = self.config['MAX_CONSOLES_SIZE_MB']
+        if self.max_consoles_size_MB:
```

Comment:
all this logic should happen in config.py
this file should just use the value in self.max_console_size